server/src/main/scala/za/co/absa/atum/server/api/controller/FlowController.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.server.api.controller
+
+import za.co.absa.atum.model.dto.{CheckpointQueryDTO, CheckpointDTO}
+import za.co.absa.atum.server.model.ErrorResponse
+import zio.IO
+import zio.macros.accessible
+
+@accessible
+trait FlowController {
+  def getFlowCheckpoints(checkpointQueryDTO: CheckpointQueryDTO): IO[ErrorResponse, Seq[CheckpointDTO]]
```

Comment:
I noticed we don't have any documentation in our traits for Controllers/Services/Repositories. I am wondering whether it would be useful to introduce it ....

Reply:
I don't know. I find it too obvious, naming and implementation wise. If you disagree let's initiate the conversation about it on our Curation tech meeting / chat though :)

Reply:
Let's bring this topic up on one of upcoming team meetings then.

Reply:
okay